lib/rbac/filterer.rb

```diff
@@ -52,12 +53,14 @@ class Filterer
       OrchestrationTemplate
       PxeImage
       ResourcePool
+      PxeServer
```

Comment:
Minor nit, but can you please move `PxeServer` above `ResourcePool` to keep it in alphabetical order. Thanks

Reply:
@ZitaNemeckova can you make this change so we can get this PR merged?

Reply:
Done :)